src/_pytest/fixtures.py

```diff
@@ -1317,7 +1379,9 @@ def _getautousenames(self, nodeid):
                 autousenames.extend(basenames)
         return autousenames
 
-    def getfixtureclosure(self, fixturenames, parentnode, ignore_args=()):
+    def getfixtureclosure(
+        self, fixturenames: Tuple[str, ...], parentnode, ignore_args: Sequence[str] = ()
```

Comment:
`parentnode: nodes.Node`

Reply:
I'm not sure, I prefer to leave this one.